utils/gen_utils.py

```diff
@@ -0,0 +1,33 @@
+"""General utility methods."""
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+class classproperty:
```

Comment:
Where do we use this? Ideally, we should change the docstring in this method to be reflective of the datacheck repo, not EvalML, but we can file an issue later on to address that once the POC is finished

Reply:
Used in data_check_action_code, data_check_action_option, data_check, and problem_types. I've changed the docstring to reflect the datacheck repo.